config/locales/partners/en/honors/honors.en.yml

```diff
@@ -260,7 +260,7 @@ en:
       label: Thesis Supervisor
       list:
         label: Thesis Supervisors
-        thesis:
+        master_thesis:
```

Comment:
That's what I was expecting too, but when I tested it out, the slug for degree type was `master_thesis` so only this translation was returning anything. I could put thesis back in if you think some slugs will read `thesis`?

Reply:
I think this is because the `slug` is derived from a database value:

```
def self.seed
    NAMES[current_partner.id].each do |name|
      d = find_or_create_by!(slug: name.parameterize.underscore) do |new_degree|
        new_degree.name = name
      end

      d.update(name:) if d.persisted?
    end
  end
```

So, the database was probably seeded with graduate data.